README.md

```diff
@@ -0,0 +1,89 @@
+# it-multistream-select
+
+## "Dialer"
+
+```js
+const pipe = require('it-pipe')
+const MSS = require('it-multistream-select')
+const Mplex = require('it-mplex') // https://github.com/alanshaw/it-mplex
+
+const muxer = new Mplex()
+const muxedStream = muxer.newStream()
+
+// MSS.select(stream, protocol(s))
+// Select from one of the passed protocols (in priority order)
+// Returns selected stream and protocol
+const { stream: dhtStream, protocol } = await MSS.select(muxedStream, [
+  // This might just be different versions of DHT, but could be different impls
+  '/ipfs-dht/2.0.0', // Most of the time this will probably just be one item.
+  '/ipfs-dht/1.0.0'
+])
```

Comment:
The multiprotocol selection is nice. Ideally the future multistream protocol will be able to avoid this, but allowing users to not have to worry about it will be nice. This also relates to the proposed `dialBest` method on libp2p https://github.com/libp2p/js-libp2p/issues/376.

Reply:
The original intent was to support semver all together. We just happened to implement the exact string matching only. But semver can still come :)